lib/devices/ios/ios.js

```diff
@@ -1036,6 +1036,18 @@ IOS.getDeviceStringFromOpts = function (opts) {
       'iPad Simulator (9.0)': 'iPad 2 (9.0)',
       'iPad Simulator (9.1)': 'iPad 2 (9.1)',
       'iPad Simulator (9.2)': 'iPad 2 (9.2)',
+      'iPhone Simulator (8.1)': 'iPhone 6s (8.1)',
```

Comment:
This hash just specifies the default, so each type should only be in once.

iPhone Simulator for 8.1-8.4 are already defined above.